start-page/style.css

```diff
@@ -56,8 +56,12 @@ body, html {
     overflow: overlay;
 }
 #shortcutscontainer1-nonium {
+    scrollbar-color: white rgba(255, 255, 255, 0.6);
     overflow-x: auto;
-    overflow-y: none;
+    overflow-y: hidden;
```

Comment:
Just a fix unrelated to the PR that I thought I'd add in to save committing later.